app/gamelib/src/game_state/changes/mod.rs

```diff
@@ -7,7 +7,6 @@ use controls::Controls;
 ///
 /// Its fields are public so they can be filled in from outside the library
 /// (before `GameState::update` is called)
-#[derive(Debug, Default)]
 pub struct Changes {
```

Comment:
Why remove the debug, default?

Reply:
Because the motion class can't be easily printed, and debug requires this

Reply:
Alternatively, what about adding a Debug implementation to `Motion`?